### PR TITLE
Guideline about our Competencies

### PR DESCRIPTION
**Pull request by Jamie-Gibson**
In the discussion about hiring guidelines, David and I talked about creating a guideline for our core competencies to help with writing descriptions (helps describe the qualifications we're looking for) hiring decisions (how close do they fit the qualities we look for) and personal development too. 

Any comments welcome - perhaps we can close by Friday 22nd Feb if there are no objections?

https://github.com/Vizzuality/playbook/blob/competencies/guidelines/competencies.md

### CODE REVIEW

File: guidelines/competencies.md [hunk 1]
```diff
@@ -0,0 +1,147 @@
+# **Vizzuality's Core Competencies**
+
+# 1. Core Competencies
+These competencies are for everybody in the team. Examples will be given for expected behaviors at each for each level
+
+## Teamwork and Collaboration 
+Working cooperatively  a team, being aware of others’ concerns and communicating this awareness empathetically. Developing partnerships with others inside and outside the organization who can provide information, assistance, and support.
```
Comment (simaob) [inline guidelines/competencies.md]:
```suggestion
Working cooperatively on a team, being aware of others’ concerns and communicating this awareness empathetically. Developing partnerships with others inside and outside the organization who can provide information, assistance, and support.
```

File: guidelines/competencies.md [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+- I find ways to measure and track customer satisfaction and setup procedures to ensure high quality of work.
+
+## Impact and Results
+Generating results that align to the project and company goals by assuming responsibility for your performance and acting efficiently within the given deadlines, organize and prioritize work, and respond to conflicting needs or seek assistance when those arise.
+
+Taking responsibility for your performance, by setting clear goals and expectations, tracking progress against the goals, ensuring feedback, and addressing performance problems and issues promptly.
+
+**Step 1**
+- I use systems to organize and keep track of information or work progress that my peers, managers and clients can understand and verify.
+- I set with my manager specific, measurable goals that are realistic but challenging, with clear dates for accomplishment, and seek support in obtaining the information, resources, and training needed to accomplish my work effectively.
```
Comment (simaob) [inline guidelines/competencies.md]:
```suggestion
- I set with my guide specific, measurable goals that are realistic but challenging, with clear dates for accomplishment, and seek support in obtaining the information, resources, and training needed to accomplish my work effectively.
```

**Comment by simaob** [comment]
I think these are very good! =) And I think they will be nicely complemented by the _role's specifications for_ mentioned in the text.

**Comment by d4weed** [comment]
> I think they will be nicely complemented by the _role's specifications for_ mentioned in the text.

Speaking of which, I've created a folder called "roles" in this same PR, and added there the specs for Front-end

https://github.com/Vizzuality/playbook/blob/competencies/guidelines/roles/frontend.md

I think we also have some specs for data written somewhere, we may also want to add other areas

please check they're ok

**Comment by Jamie-Gibson** [comment]
Ok so if there's no other comments, let's push this.

**Pull request merged**